chia/full_node/full_node.py

```diff
@@ -619,6 +619,7 @@ async def short_sync_batch(self, peer: WSChiaConnection, start_height: uint32, t
                 fork_hash = self.constants.GENESIS_CHALLENGE
             assert fork_hash
             fork_info = ForkInfo(start_height - 1, start_height - 1, fork_hash)
+            blockchain = AugmentedBlockchain(self.blockchain)
```

Comment:
on line 640, we create another `blockchain = AugmentedBlockchain()`. It looks like a mistake

Reply:
right this is from the merge with my other pr that went into main, thanks